### PR TITLE
Add spanish translation of readme

### LINKED ISSUE #3106: Translation For  Readme (#HacktoberFest)
Hi. Can I make a PR for  translating this repo's readme to Hindi? It's the world's 3rd morst spoken language, maybe you would want it.
Thankyou

### LINKED ISSUE #3188: Add spanish translation readme
Add spanish translation of readme.
I want to correct them and work on this issue as of Hackoberfest.

### PR DESCRIPTION
**Pull request by parammittal16**
### Affected components 

- Docs

### Related Issues

Fixes #3106  and #3188 

### Solution and Design

Add spanish translation of readme

### CODE REVIEW

**Comment by parammittal16** [comment]
@shahidhk plz review sir

File: translations/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) | [:es: española](README.spanish.md)          ￨
```
Comment (Skao93) [inline translations/README.md]: Acá debería decir "Español" en vez de "española"

File: translations/README.spanish.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+### Datos derivados o transformaciones de datos
+
+Transforme los datos en Postgres o ejecute la lógica de negocios en ellos para derivar otro conjunto de datos que se pueda consultar con GraphQL Engine - [leer más] (https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html )
+
+## Demos
+
+Consulte todas las aplicaciones de ejemplo en
+Directorio [community / examples] (community / examples).
+
+### aplicaciones en tiempo real
```
Comment (Skao93) [inline translations/README.spanish.md]: Este título necesita mayúsculas iniciales

File: translations/README.spanish.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+[! [CircleCI] (https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)] (https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"> <img src = "https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"> </ a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"> <img src = "https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo = twitter "> </a>
+<a href="https://eepurl.com/dBUfJ5"> <img src = "https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"> </a>
+
+Hasura GraphQL Engine es un servidor GraphQL ultrarrápido que le ofrece ** API GraphQL instantáneas y en tiempo real sobre Postgres **, con [** activadores de webhook **] (event-triggers.md) en eventos de base de datos y [** remoto esquemas **] (remote-schemas.md) para lógica de negocios.
+
+Hasura lo ayuda a construir aplicaciones GraphQL respaldadas por Postgres o moverse gradualmente a GraphQL para aplicaciones existentes que usan Postgres.
```
Comment (Skao93) [inline translations/README.spanish.md]: Es más común utilizar "migrar" que "moverse", y en el caso de "backed by" creería que es mejor decir "…aplicaciones GraphQL sobre PostgreSQL…"

File: translations/README.spanish.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+Esto incluye todo en [`docs`] (docs) y [` community`] (community)
+directorios.
+
+## Traducciones
+
+Este archivo Léame está disponible en las siguientes traducciones:
+
+- [Japonés: jp:] (traducciones / README.japanese.md) (: orar: [@moksahero] (https://github.com/moksahero))
+- [Francés: fr:] (traducciones / README.french.md) (: orar: [@ l0ck3] (https://github.com/l0ck3))
+- [Español: es:] (traducciones / README.spanish.md) (: orar: [@ parammittal16] (https://github.com/parammittal16))
```
Comment (Skao93) [inline translations/README.spanish.md]: Estos emojis traducidos no funcionan. Sospecho que esta es una traducción de Google Translate. 

Is this a Google Translate text? Looks like one, you even translated the emojis.

**Comment by parammittal16** [comment]
@Skao93 mam plz review I apologize for the mistakes but now I have corrected them plz review

File: translations/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) | [:es: Español](README.spanish.md)          ￨
```
Comment (marionschleifer) [inline translations/README.md]: This doesn't show up correctly yet:

image

File: translations/README.spanish.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+## Tabla de contenido
+<!-- markdown-toc start - No edite esta sección. Ejecute M-x markdown-toc-refresh-toc -->
+**Tabla de contenido**
+
+- [Inicio rápido:](#inicio-rápido)
+    - [Implementación con un clic en Heroku](#despliegue-con-un-clic-en-heroku)
+    - [Otros métodos de implementación](#otros-métodos-de-implementación)
+- [Arquitectura](#arquitectura)
+- [Herramientas del lado del cliente](#herramientas-del-lado-del-cliente)
+- [Agregar lógica de negocios](#añadir-lógica-de-negocios)
```
Comment (marionschleifer) [inline translations/README.spanish.md]: Here it's `Agregar` and below in the title it's `Añadir`. Could you fix this please?

File: translations/README.spanish.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+procesamiento - [leer más](../event-triggers.md).
+
+### Datos derivados o transformaciones de datos
+
+Transforme los datos en Postgres o ejecute la lógica de negocios en ellos para derivar otro conjunto de datos que se pueda consultar con GraphQL Engine - [leer más](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demos
+
+Consulte todas las aplicaciones de ejemplo en
+Directorio [community/examples](../community/examples).
```
Comment (marionschleifer) [inline translations/README.spanish.md]: This link is broken. Could you add the correct one please?

File: translations/README.spanish.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+
+### Aplicaciones en tiempo real
+
+- La aplicación Group Chat creada con React, incluye un indicador de escritura, usuarios en línea y nuevos
+  notificaciones de mensajes.
+  - [Pruébalo](https://realtime-chat.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-chat)
```
Comment (marionschleifer) [inline translations/README.spanish.md]: This link doesn't work. Could you fix it?

File: translations/README.spanish.md [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+  - [Examinar API](https://realtime-chat.demo.hasura.app/console)
+
+- Aplicación de seguimiento de ubicación en vivo que muestra un vehículo en marcha cambiando el GPS actual
+  coordenadas moviéndose en un mapa.
+  - [Pruébalo](https://realtime-location-tracking.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-location-tracking)
```
Comment (marionschleifer) [inline translations/README.spanish.md]: This link is not working. Could you fix it? I think if you merge the latest changes from Master, it should be fixed.

File: translations/README.spanish.md [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+  - [Examinar API](https://realtime-location-tracking.demo.hasura.app/console)
+ 
+- Un panel de control en tiempo real para agregaciones de datos en datos que cambian continuamente.
+  - [Pruébalo](https://realtime-poll.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-poll)
```
Comment (marionschleifer) [inline translations/README.spanish.md]: Same here :)

**Comment by marionschleifer** [comment]
@Skao93 thank you so much for reviewing this 🙏 could you give it another review with the changes @parammittal16 has implemented? That would be amazing 🙌

File: translations/README.spanish.md (hunk 10 above)
Comment (parammittal16) [reply translations/README.spanish.md]: Done mam

File: translations/README.spanish.md (hunk 9 above)
Comment (parammittal16) [reply translations/README.spanish.md]: Done mam

File: translations/README.spanish.md (hunk 8 above)
Comment (parammittal16) [reply translations/README.spanish.md]: Done mam

File: translations/README.spanish.md (hunk 7 above)
Comment (parammittal16) [reply translations/README.spanish.md]: Mam in latest master these does not exist 
I think examples folder has been removed

File: translations/README.spanish.md (hunk 6 above)
Comment (parammittal16) [reply translations/README.spanish.md]: Done mam

File: translations/README.md (hunk 5 above)
Comment (parammittal16) [reply translations/README.md]: Done mam

**Comment by parammittal16** [comment]
Now i think all the changes are done and it can be merged A lot of hard work :) @marionschleifer @Skao93 plz review

**Comment by shahidhk** [comment]
@parammittal16 Can you confirm if you have used Google translate for this?

**Comment by marionschleifer** [comment]
@Skao93 thank you for reviewing our Spanish PRs 🙂 could you please send an email to marion@hasura.io, so that we can send you some swag as a thank you?

**Pull request closed without merging**